flink-runtime/src/main/java/org/apache/flink/runtime/state/FullSnapshotUtil.java

```diff
@@ -16,13 +16,13 @@
  * limitations under the License.
  */
 
-package org.apache.flink.contrib.streaming.state.snapshot;
+package org.apache.flink.runtime.state;
 
 /**
- * Utility methods and constants around RocksDB creating and restoring snapshots for {@link
- * org.apache.flink.contrib.streaming.state.RocksDBKeyedStateBackend}.
+ * Utility methods and constants around creating and restoring full snapshots using {@link
+ * FullSnapshotAsyncWriter}.
  */
-public class RocksSnapshotUtil {
+public class FullSnapshotUtil {
 
     /** File suffix of sstable files. */
     public static final String SST_FILE_SUFFIX = ".sst";
```

Comment:
nit: This constant is Rocks specific, isn't it?

Reply:
True, I will have to move some of them back to a Rocks-specific `RocksSnapshotUtil`.